setup.cfg

```diff
@@ -7,3 +7,12 @@ exclude = venv*,__pycache__,node_modules,cache,migrations,build,sample_cap_xml_d
 max-line-length = 120
 # W504 line break after binary operator
 extend_ignore=B306, W504
+
+
+[isort]
+line_length=80
```

Comment:
do we want line length of 80 when our usual limit is 120?

Reply:
I guess if we use the same in admin it should be alright

Reply:
we should have it as 120 in both i think.

Reply:
I chose 80 originally. I try to write code to 80 characters, but sometimes there’s a good reason to make lines a bit longer, which is why our flake8 limit is set to 120. Somewhere in this [excellent Raymond Hettinger talk](https://www.youtube.com/watch?v=wf-BqAjZb8M) he talks about not being overly-dogmatic about line length, at the expense of readability.

An automated tool can’t make the same judgement calls about when it’s sensible to exceed 80 characters, so I don’t think we should let it.

Reply:
I like 120 limit more, as it allows for more descriptive method and variable names. I think 80 chars limit was set as a standard when screens had much lower resolution, right? 

Still, here it's not that important to me since it's just imports, so I'm happy either way ☺️

Reply:
I agree with @quis. If we're going to have a limit, then it doesn't make much difference what it is, since the output is basically automatic.